src/frontend/src/store/modules/recommendations.js

```diff
@@ -47,14 +35,15 @@ export default {
     },
     async ON_QUESTION_RECOMMENDATION({ commit }, payload) {
```

Comment:
이렇게 받으면 아래 putAction body 부분에 `{ recommendationType }` 으로만 써도 되겠네요
```suggestion
    async ON_QUESTION_RECOMMENDATION({ commit }, {recommendationType}) {
```